tools/oot-driver/hack/create-special-resource.sh

```diff
@@ -25,6 +28,8 @@ do
       then
         sed 's/name: \"oot-source-driver:latest\"/name: \"\$\{INTERNAL_REGISTRY\}\/oot-source-driver:latest\"\n          pullsecret: external-registry/g' -i ./special-resource.yaml
         sed 's/ImageStreamTag/DockerImage/g' -i ./special-resource.yaml
+        envsubst < "./special-resource.yaml" > temp
```

Comment:
why we need to run this one again? it's already in line 26

Reply:
there is a variable inside the `sed` command that I could not replace so I did a dirty fix and used `envsubst` again. However I fixed this using a better `sed` command